src/components/Link.utils.tsx

```diff
@@ -46,7 +48,54 @@ export function IconExternalLink(): ReactElement {
   );
 }
 
-export function isNewTabImplied(href: string | undefined): boolean {
+// Determine if the the target href should open in a new tab
+export function useIsNewTabImplied(href: string | undefined): boolean {
```

Comment:
NIT: Could you do me another favor here and just add like a 2 line comment explaining why we're doing it this way for our future selves? Something like, determines if a link should be opened in a new tab based on design team's rules to avoid bespoke logic per link?

Reply:
Comment updated.